### DISCUSSION TITLE
Missing several translations

### DISCUSSION THREAD

**Discussion by Piepe6213** [Leantime/leantime#1274] [General] [answered]
Hi,

I tried "de-DE_informal" an "de-DE", there several translations missing. I looked up in the language-inis but can't find any corresponding entry. Is leantime not fully translatable?
Thanks!

**Comment by marcelfolaron** [comment, marked as answer]
Hi,

Leantime is being translated by the community as best as possible. With
that said most translations are running behind.
Any contributions to the translations are very welcome.

You can see that status of the current translations here:
https://crowdin.com/project/leantime
